pandas/io/common.py

```diff
@@ -435,7 +444,13 @@ class BytesZipFile(zipfile.ZipFile, BytesIO):  # type: ignore
     """
 
     # GH 17778
-    def __init__(self, file, mode, compression=zipfile.ZIP_DEFLATED, **kwargs):
+    def __init__(
+        self,
+        file: Union[BinaryIO, str],
```

Comment:
Hmm did FilePathOrBuffer not work here? Does ZipFile not accept pathlib.Path objects?

Reply:
it works. will change.